mod_sample/media_info_parser.py

```diff
@@ -212,13 +215,9 @@ def generate_media_xml(sample) -> MediaInfoFetcher:
         process = subprocess.Popen(['mediainfo', '--Output=XML', media_path], stdout=output_handle)
         process.wait()
         if os.path.isfile(media_info_path):
-            # Load media info, and replace full pathname
+            # Load media info
             tree = etree.parse(media_info_path)
-            for elem in tree.xpath("//track[@type='General']"):
-                for child in elem:
-                    if child.tag == 'Complete_name':
-                        child.text = child.text.replace(media_folder, '')
-                        break
```

Comment:
Why was this deleted? We don't want to leak path names of the file to persons who download the XML.

Reply:
There is no such tag as "Complete_name" in the XML generated by the latest version of mediainfo.

Reply:
It's now located in the Media ref attribute. The thing we want to achieve is replacing the full filename (including directories) with just the filename.

Reply:
Yes, I just made a comment on that in the below part of the code :)